<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgProofingOptions</class>
 <widget class="QWidget" name="WdgProofingOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="KisColorSpaceSelector" name="proofSpaceSelector" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Rendering Intent</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QComboBox" name="cmbIntent">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="ckbBlackPointComp">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Black Point compensation matches the darkest color of the source device to the darkest color of the destination device. Relative Colorimetric without Black Point Compensation will show the difference between the darkest values. With blackpoint compensation, black is black.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Black Point Compensation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Display Mode:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbDisplayTransformState">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="grbDisplayConversion">
     <layout class="QFormLayout" name="formLayout_3">
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Rendering Intent:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cmbDisplayIntent"/>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="chkDisplayBlackPointCompensation">
        <property name="text">
         <string>Black Point Compensation</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="chkDoChromaticAdaptation">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enables chromatic adaptation for absolute rendering intent. Effectively this will show how the colors would look if the image was printed on a paper and viewed with the monitor as the only light source. This is standard behaviour for a ICC v4 workflow.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Absolute Intent Chromatic Adaptation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lblGamutWarning">
       <property name="text">
        <string>Gamut Warning:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KisColorButton" name="gamutAlarm">
       <property name="toolTip">
        <string>Set color used for warning</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>63</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisColorButton</class>
   <extends>QPushButton</extends>
   <header>kis_color_button.h</header>
  </customwidget>
  <customwidget>
   <class>KisColorSpaceSelector</class>
   <extends>QWidget</extends>
   <header>widgets/kis_color_space_selector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
